tests/test_cli.py

```diff
@@ -208,13 +211,15 @@ def test_lang_eng(self) -> None:
             "spaces test.pdf",
         ],
     )
+    @pytest.mark.skipif(dummy_conversion(), reason="Won't run with dummy converter")
```

Comment:
I believe that skipping these tests will not be necessary once we fix https://github.com/freedomofpress/dangerzone/pull/302#discussion_r1068369476. But let's discuss more on that thread.